job-dsls/jobs/prod/prod_cron_meta_nightly_pipeline.groovy

```diff
@@ -60,7 +60,7 @@ pipeline{
         stage('trigger KOGITO nightly job ${KOGITO_NEXT_PRODUCT_BRANCH}') {
             steps {
                 build job: 'kogito.nightly/${KOGITO_NEXT_PRODUCT_BRANCH}', propagate: false, wait: true, parameters: [
```

Comment:
Isn't the issue also here?
Based on the [latest run](https://eng-jenkins-csb-business-automation.apps.ocp-c1.prod.psi.redhat.com/blue/organizations/jenkins/PROD%2Fcron-meta-nightly-pipeline/detail/cron-meta-nightly-pipeline/342/pipeline) it tried to run `kogito.nightly/7.67.x`, which doesn't exist of course. The issue is probably that `KOGITO_NEXT_PRODUCT_BRANCH` is the same as `NEXT_PRODUCT_BRANCH`:
https://github.com/kiegroup/kie-jenkins-scripts/blob/eac8b77f918aa963cbb2ad993c2188b4700b908c/job-dsls/jobs/prod/prod_cron_meta_nightly_pipeline.groovy#L20
which is set to `BRANCH`:
https://github.com/kiegroup/kie-jenkins-scripts/blob/eac8b77f918aa963cbb2ad993c2188b4700b908c/job-dsls/jobs/prod/prod_cron_meta_nightly_pipeline.groovy#L12
which is set to 7.67.x:
https://github.com/kiegroup/kie-jenkins-scripts/blob/eac8b77f918aa963cbb2ad993c2188b4700b908c/job-dsls/src/main/groovy/org/kie/jenkins/jobdsl/Constants.groovy#L25

Reply:
I am actually fixing this in another PR (in progress) with some refactoring to make the pipeline definition less error-prone. I will post the PR here in a while.

Reply:
https://github.com/kiegroup/kie-jenkins-scripts/pull/1234